raiden-network/light-client
# Feature/dapp planned udc withdrawal extension

**Thank you for submitting this pull request :)**

Fixes #2308

**Short description**
The first two commits includes minor issues which have produced noise.. Sorry for that.
Well, then go for it and do a withdraw and wait a while. I did it myself and it worked for me.
Note that while the new component from the last commit follows 100% the linked design model of Sash, the width inside the UDC screen is not yet correct. This is part of #2220 that reworks the structure and design of this screen.

**Definition of Done**


**Steps to manually test the change (dApp)**

1. Go to the UDC screen of your account
2. Verify that you have no planned withdraws
3. Initiate a withdraw
4. Verify that you see a pending planned withdraw with down counting remaining blocks
5. Wait that the planned withdraw becomes ready and changes its appearance
6. Follow the link to actually withdraw the tokens
7. Go back to the UDC screen and verify that there is no planned withdrawal anymore (this eventually needs 1-2 blocks more to happen, since the blockchain events are not yet seen)
